Formula/pokerstove.rb

```diff
@@ -0,0 +1,35 @@
+class Pokerstove < Formula
+  desc "Poker evaluation and enumeration software"
+  homepage "https://github.com/andrewprock/pokerstove"
+  url "https://github.com/andrewprock/pokerstove/archive/v1.0.tar.gz"
+  sha256 "68503e7fc5a5b2bac451c0591309eacecba738d787874d5421c81f59fde2bc74"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      bin.install "bin/ps-eval"
+      bin.install "bin/ps-lut"
+      bin.install "bin/ps-colex"
+      bin.install "bin/penum_tests"
+      bin.install "bin/peval_tests"
+      bin.install "bin/util_tests"
+    end
+  end
+
+  def caveats; <<~EOS
+    To run:
```

Comment:
Is this specific to the macOS install?

Reply:
Not specific.

Reply:
There is a need to do this since the package name `pokerstove` and the executable file `ps-eval` is a hard relating command name.  Without, some messages users might have to hard time finding the command `ps-eval`.

Reply:
Upstream doc is pretty clear:

<img width="939" alt="Capture d’écran 2019-08-26 à 17 42 53" src="https://user-images.githubusercontent.com/1980544/63703320-097e5780-c829-11e9-8f0d-71415d320fc3.png">

So let's remove the `caveats`.